xbee/thread/base.py

```diff
@@ -20,6 +20,9 @@
 class ThreadQuitException(Exception):
     pass
 
+class TimeoutException(Exception):
```

Comment:
This should move up into the backend folder to be shared across implementations.